ResearchKit/Common/ORKGraphConstants.h

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
The contents of this file should probably be part of `ORKBaseGraphView_Internal.h`, instead of standalone.